_summaries/wang2018dynamic_1.md

```diff
@@ -1,19 +1,95 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: wang2018dynamic
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: zayne-sprague
+score: 8
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+## The Core Idea of Dynamic Graph CNN
+
+- By utilizing edge features of nearby vertices connecting to another vertex, we can apply CNN operations on them computing local and global feature maps for the point cloud that is invariant to permutation, has local structure, and is easy to implement.
+
+## What are edges in a point cloud?
+
+- Graphs are defined by a local neighbourhood of vertices found by using KNN (which points are closest to the current vertex in question)
+
+- Edges are constructed by drawing directed edges to each vertex in the local graph, including edges pointing to the same vertex.
+    - This should sound familiar -- it's similar to self-attention from transformers 
+  
+![Edge Construction](wang2018dynamic_1b.png)
+
+## How do we calculate the values of these edges (how is it realized technically)
+
+- These edges are used as features for the DGCNN by running the vertices through a fully connected layer (or layers) multiple times
+
+- The feature for a vertex is an aggregation function applied to all the edge features (the convolution operation)
+
+- You can change the non-linearity and the aggregation function in different ways to achieve various effects.
+
+![Edge Conv](wang2018dynamic_1c.png)
+  - Look at how $$e_{ij}$$ is computed, you take both vertices and feed their features (in this case X,Y,Z coordinates) into linear layers that will be convoled over later
```

Comment:
I think this diagram either requires more explanation, or could be replaced with something else.  I'm thinking maybe Table 1 from the paper.  Once you add the formal definition of EdgeConv, I think Table 1 is pretty easy to understand, and it really clearly shows in way sense EdgeConv is "both global and local".  Another added benefit is that the edge function's dependence only on `x_i` or `x_j`  for PointNet variants really shows how localized PointNet operations are.

Reply:
Sounds good, I'll add Table 1 here